build.ps1

```diff
@@ -52,6 +52,13 @@ try {
         Get-ChildItem -Path $PSScriptRoot\bench -Filter "BenchmarkDotNet.Artifacts" -Directory -Recurse | Move-Item -Destination "$PSScriptRoot\artifacts\benchmarks"
     }
 
+    if ($env:CI -eq "True" -or $env:CI -eq "true") {
```

Comment:
```suggestion
    if ($env:CI -eq "true") {
```

PowerShell is case-insensitive by default.

Reply:
Ah, need more practice with Powershell. More the bash type of person 😄 

Cheers